api/README.md

```diff
@@ -14,17 +14,20 @@ to start database and `outpack_server` instances.
 
 ## Starting App
 
-There are two ways to start the app:
+Before running the app you need to configure it according to the auth method you wnat to use. The support auth methods are:
 
-1. The default dev configuration enables Github auth, but you will need to set environment variables PACKIT_GITHUB_CLIENT_ID
+- **github**: The default dev configuration enables Github auth, but you will need to set environment variables PACKIT_GITHUB_CLIENT_ID
    and
    PACKIT_GITHUB_CLIENT_SECRET with the details of an OAuth app which you can use to log in. You can use the details held in
    the
-   mrc vault at `VAULT:secret/auth/githubclient/id:value` and `VAULT:secret/auth/githubclient/secret:value`.
-2. The other configuration is to use basic auth. This is enabled by setting `auth.method=basic` in `application.properties`.
-3. No auth mode. This is enabled by setting `auth.enabled=false` in `application.properties`.
-   Run up app on command line from the /api directory
-   `./scripts/run-dev`
+   mrc vault at `VAULT:secret/auth/githubclient/id:value` and `VAULT:secret/auth/githubclient/secret:value`. 
```

Comment:
I think this should be updated to say

```
You can use the details held in the mrc vault at `secret/packit/githubauth/auth/githubclient`.
```

since the paths listed don't seem to exist.

Reply:
Or we can encourage people to use some `dev` details in case the ones I just referred to are tied to a non-dev app - seems risky to mix up prod and dev secrets.

Here are some dev details:

`vault read secret/packit/oauth/dev`

^ but maybe this 'oauth' folder is liable to be deleted at some point given we do not support oauth.

Reply:
yep, good spot, it had been updated in auth.md but not here. We probably don't need this many READMEs about auth!

I made a ticket to consolidate some of this stuff: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-6452/

"oauth" here refers to authenticating with GitHub using its oauth path. Could have been better named perhaps. I think at one stage we were imagining supporting other OAuth providers so "oauth" and "github" authenticating got a bit conflated.